.prettierignore.json

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
Instead of ignoring, let's use a more specific pattern in package.json. something like `{src,stories,test}/**/*.{js,md,yml}`. We don't want to accidentally format stuff like the lockfile or node modules.

Reply:
Prettier automatically ignores node_modules by default.

I'd rather just add `package-lock.json` to the ignore file than a more specific targets. Otherwise the specific pattern would be `.github,.stories,src,stories,test` plus all the files in the root directory like the readme or jest config. 

For the time being I'm just adding an extra ignore for `package-lock.json`, let me know if you disagree.